install/tools/istio_vm_common.sh

```diff
@@ -223,11 +225,30 @@ function istioCopyBuildFiles() {
      $ISTIO_IO/pilot/bazel-bin/tools/deb/istio-agent.deb \
      $ISTIO_IO/auth/bazel-bin/tools/deb/istio-auth-node-agent.deb \
      $ISTIO_IO/istio/install/tools/istio_vm_setup.sh \
-     $ISTIO_FILES
+     $ISTIO_STAGING
   # For override to work
   chmod +w *.deb
 }
 
+# Download the istio release files into ISTIO_STAGING
+# ISTIO_VERSION will be used.
+function istioDownloadVMFiles() {
+  local ISTIO_IO=${ISTIO_BASE:-${GOPATH:-$HOME/go}}/src/istio.io
+  local ISTIO_STAGING=${ISTIO_STAGING:-$ISTIO_IO/istio/bazel-bin/vm}
+  local ISTIO_VERSION=${ISTIO_VERSION:-0.2.3}
```

Comment:
can we not hard code version here?

Reply:
I like to have defaults - but I could source istio.version file ( since this will be part of the samples ).

Reply:
downloadIstio.sh  (https://git.io/getIstio) also hardcodes the version, there is a TODO to update it automatically
one each release - when this is done we'll update here as well.

( in any case - removed this code, it is now part of downloadIstio )